rtloader/include/datadog_agent_rtloader.h

```diff
@@ -29,24 +29,26 @@ struct rtloader_pyobject_s;
 typedef struct rtloader_pyobject_s rtloader_pyobject_t;
 
 // FACTORIES
-/*! \fn rtloader_t *make2(const char *python_home, char **error)
+/*! \fn rtloader_t *make2(const char *python_home, const char *python_exe, char **error)
```

Comment:
Yeah I thought about this option for a while too but there's a problem of consistency since:
-This can easily be confused with the python interpreter's `bin` folder that holds the executable instead of the abs path to the interpreter binary itself.
- `exec`/`executable`([link](https://docs.python.org/3/c-api/init.html#c.Py_GetExecPrefix), [link](https://docs.python.org/3/c-api/init.html#c.Py_SetPath)) is the terminology used in the Python C API to indicate the path to the Python interpreter.

So I thought the lower chance of confusion would be to apply the C API common pattern. Maybe we can do `python_exe` -> `python_exec` instead? I don't have a strong opinion either way though so up to you how you would like this fixed up.

Reply:
You'r right, `python_exe` is better then.